core/src/main/java/org/elasticsearch/index/query/TemplateQueryParser.java

```diff
@@ -67,28 +51,18 @@ public TemplateQueryParser(ScriptService scriptService) {
      * values. Handles both submitting the template as part of the request as
      * well as referencing only the template name.
      *
-     * @param context
+     * @param parseContext
```

Comment:
or simply remove the line. the param tag by itself is useless without a description